### PR TITLE
new blueshield backpacks (sprites by Rapaskoti)

### PR DESCRIPTION
**Pull request by Aquilar**
## What Does This PR Do

adds a set of backpacks to the blueshield, replacing the security ones they currently have.

this was done for Rapaskoti as they couldn't get the new sprites to appear in game.

## Images of changes

image
image

## Changelog
:cl:
add: Blueshield gets new backpacks

/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 1]
```diff
@@ -287,6 +293,11 @@
 	icon_state = "satchel-cap"
 	resistance_flags = FIRE_PROOF
 
+/obj/item/storage/backpack/satchel_blueshield
+	name = "blueshield satchel"
+	desc = "A robust satchel issue to Nanotrasen's finest."
```
Comment (farie82) [inline code/game/objects/items/weapons/storage/backpack.dm]: Issued

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 2]
```diff
@@ -531,6 +542,12 @@
 	icon_state = "duffel-clown"
 	item_state = "duffel-clown"
 
+obj/item/storage/backpack/duffel/blueshield
+	name = "blueshield duffelbag"
+	desc = "A robust duffelbag issue to Nanotrasen's finest."
```
Comment (farie82) [inline code/game/objects/items/weapons/storage/backpack.dm]: Issued

**Comment by Necaladun** [comment]
Simple but effective, love it.

**Comment by shawnhind** [comment]
Looks good.  Excited to see this get merged

**Pull request merged**